config/example.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
-export DM_API_DOMAIN=${DM_API_DOMAIN:=http://localhost:5000}
+export DM_ENVIRONMENT=${DM_ENVIRONMENT:="preview"}
```

Comment:
As discussed offline - not sure about this. I'd rather have a separate `config/preview.sh.example` file or better yet, get rid of this example file entirely and find a way to plug in the creds repo to run locally with minimal faff (something for firebreak, maybe...)

Reply:
Yup, maybe, looks like all of us use this file slightly differently. Guess what we could do is have a `common` file which is basically the same as this file and a `staging.sh.example` which simply defines `DM_ENVIRONMENT` then includes `common` and a `local.sh.example` which overrides a number of things (then includes `common`)... though it all feels a lot like going back to how it *was*.